templates/confidential-computing/ai.html

```diff
@@ -0,0 +1,246 @@
+
+        </div>
+
+        <hr class="p-rule is-fixed-width is-muted" />
+
+        <div class="row">
+
+          <div class="col-3 col-medium-2">
+
+            <h3 class="p-heading-icon__title p-heading--5">Securely integrated CPU-GPU solution</h3>
+
+          </div>
+
+          <div class="col-6 col-medium-3">
+
+            <p>
+              The CPU-GPU communication is encrypted, and a strong cryptographic guarantee is available for all remote parties to remotely attest the security claims of the platform.
+
+            </p>
+
+            <hr class="p-rule is-fixed-width is-muted" />
+
+            <p>
+              <a href="/confidential-computing">Learn more about confidential computing&nbsp;&rsaquo;</a>
+            </p>
+
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Confidential AI with Ubuntu and Nvidia&nbsp;H100&nbsp;GPUs</h2>
+      </div>
+      <div class="col">
+        <p>
+          Enterprises using machine learning in the cloud are concerned about the security of their data and the protection of their models. Industry regulations often prevent sharing sensitive data, hindering AI's full potential in important fields.
+        </p>
+        <p>
+          To address these challenges on Azure, a confidential AI preview is available with Ubuntu confidential VM using AMD 4th Gen EPYC processors with SEV-SNP, alongside  NVIDIA H100 GPUs.
+        </p>
+        <div class="p-section--shallow">
+          <div class="u-aspect-ratio--16-9 u-background--fade u-vertically-center u-align--center">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/fc0006be-cvm-diagram@2x.png",
+            alt="Ubuntu confidential VMs establish a secure encrypted communication channel with Nvidia H100 GPUs",
+            width="600",
+            height="338",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+        </div>
+        <hr class="p-rule is-muted" />
+        <p>
+          <a href="https://aka.ms/accgpusignup">Sign up for the Azure preview of confidential AI with Ubuntu&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Use Ubuntu Pro to further harden your confidential&nbsp;VMs</h2>
+      </div>
+      <div class="col">
+        <p>
+          While confidential VMs can protect your workload from external threats, vulnerabilities from within their boundaries remain a concern. This is where Ubuntu Pro becomes essential, to keep your guest CVM software stack always patched and up-to-date.
+        </p>
+
+        <hr class="is-muted" />
+        <a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
```

Comment:
```suggestion
        <p><a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a></p>
```
Flagging since I've gotten this feedback from Lyubo, links should be wrapped in paragraphs